andela/reaction-commerce
# #133002639 admin can manage static pages

### What does this PR do?
Adds the capability for the shop admin users to create, edit and delete static pages in the application. Non-admin users can only view these static pages.

### Description of Task to be completed?
* Add a plugin to the dashboard so the admin can view static pages
* Add a pages plugin that displays the static pages
* Add default data to be load on application startup
* Create new `Pages` collection and schema to hold and access the static page data

### How should this be manually tested?
On application startup, a non-admin user should see the `other pages` dropdown menu on the navigation bar and clicking on it should reveal the list of static pages available. Admin users can signin and create, edit and delete static pages by going to the `static pages` addon on the dashboard.

### What are the relevant pivotal tracker stories?
#133002639 Static Page Management - Admin can manage static pages
